File: OOP_C++_JAVA/lab_1.cpp
#include <bits/stdc++.h>
using namespace std;
class Triangle
{
public:
    double width, height;
    Triangle(double width, double height)
    {
        this->width = width;
        this->height = height;
    }
    void area()
    {
        int result = 0.5*height*width;
        cout<<"Area of the Triangle " << result<<endl;
    }
};
int main()
{
    double h, w, a;
    cout << "Enter the Height and Width  :" << endl;
    cin >> h >> w;
    Triangle object = Triangle(h, w);
    object.area();
    return 0;
}

File: OOP_C++_JAVA/lab1.java
import java.util.Scanner;

class Triangle {
    public double height;
    public double width;

    Triangle(double height, double width) {
        this.height = height;
        this.width = width;
    }

   public void  area() {
        double result = 0.5*height*width;
        System.out.println("Area oth the Triangle : "+result);

    }

}

public class lab1 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        System.out.println("Enter the Height and width: ");
        double h = input.nextDouble();
        double w = input.nextDouble();
        Triangle T = new Triangle(h, w);
         T.area();
    
        input.close();
    }
}
